docs/getting-started.md

```diff
@@ -0,0 +1,173 @@
+- Plug a drive that contains the image you would like to flash into the slot with the blue-colored blinking LED
+- Select the image you want to flash from the file browser and select **OK**
+- Plug at least one drive or device into an available slot. The plugged target(s) will be selected automatically and the LEDs will turn white
+- Select **Flash**, to begin the flashing process
+- The LED of each slot will first blink purple for flashing and then green for validating
+- Once the flashing is complete, the LEDs will turn green for successfully flashed drives, and red for the failed ones
+- You may safely unplug the drives when flashing is complete
+
+### Flash from URL
+
+Using an online image file as source to flash to one or multiple targets
+
+- From the Etcher menu, select **Flash from URL**
+- Enter the image URL of you would like to flash in the input field, and select **OK**
+- Plug at least one drive or device into an available slot. The plugged target(s) will be selected automatically, and the LEDs will turn white
+- Select **Flash**, to begin the flashing process
+- The LED of each slot will first blink purple for flashing and then green for validating
+- Once the flashing is complete, the LEDs will turn green for successfully flashed drives, and red for the failed ones
+- You may safely unplug the drives when flashing is complete
+
+### Clone drive
+
+Using a drive as source, and cloning it to multiple drives
+
+- From the Etcher menu, select **Clone drive**
+- Plug a drive you would like to clone into the slot with the blue-colored blinking LED
+- The drive will be selected automatically and the LED will stop blinking
+- Plug at least one drive into an available slot. The plugged targets will be selected automatically, and the LEDs will turn white
+- Select **Flash**, to begin the flashing process
+- The LED of each slot will first blink purple for flashing and then green for validating
+- Once the flashing is complete, the LEDs will turn green for successfully flashed drives, and red for the failed ones
+- You may safely unplug the drives when flashing is complete
+
+### Backup drive
+
+Backing up one or multiple drives into another drive
+
+- From the Etcher menu, select **more options**, then **Backup drive**
+- Plug one or more drives you would like to backup into the slot(s) with the blue-colored blinking LED
+- The drives will be autoselected and the LED will stop blinking
+- Select **OK** to move on to the next step
+- Plug the backup drive into the slot with the white-colored blinking LED. The plugged target will be automatically selected and the LED will stop blinking
+- Select **Flash**, to begin the flashing process
+- The LED of the target slot will first blink purple for flashing, and then green for validating
+- Once the flashing is complete, the LED will turn green for a successful flash, and red for a failed one
+- You may safely unplug the drive when flashing is complete
+
+### Format drive
```

Comment:
ATM there is no format option on EtcherPro, can you remove this section please?

Reply:
Will do